spec/fixtures/modules/sample/tasks/error.ps1

```diff
@@ -0,0 +1,2 @@
+Write-Output 'error'
```

Comment:
I think these are duplicates of `modules/error/tasks/fail.sh` and `modules/error/tasks/fail.ps1` - can those be reused for the CLI tests?